src/screens/FeedScreen/elements/AddCommentPreview.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {TouchableOpacity, StatusBar} from 'react-native';
+import WriteComment from '../../../components/Comments/WriteComment';
+import {COLORS} from '../../../utils/theme';
+import {normalize} from '../../../utils/fonts';
+
+type AddCommentPreviewProps = {
+  username: string;
+  isBlurred: boolean;
+  heightReaction: number;
+  onPressComment: () => void;
+};
+
+const tabBarHeight = StatusBar.currentHeight;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `StatusBar.currentHeight` directly might lead to incorrect values on some Android devices, as it returns the height of the status bar and can return `null`. It's better to use a safer approach for getting the height of the status bar.

```diff
- const tabBarHeight = StatusBar.currentHeight;
+ import { getStatusBarHeight } from 'react-native-status-bar-height';
+ const tabBarHeight = getStatusBarHeight();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->